docker/supervisor.conf

```diff
@@ -0,0 +1,21 @@
+[supervisord]
+logfile=/code/logs/supervisord.log
+
+[program:olympia]
+command=python /code/manage.py runserver 0.0.0.0:8000
+directory=/code
```

Comment:
because of this, can't you just do `python manage.py` ? I think some pesky things in Django only work when you're in the working dir